angular/projects/spark-angular/src/lib/components/sprk-card/sprk-card.stories.ts

```diff
@@ -157,41 +338,289 @@ export const teaser = () => ({
  `,
 });
 
-teaser.story = {
-  name: 'Teaser',
+legacyTeaser.story = {
+  name: 'Legacy Teaser (Deprecated)',
   parameters: {
     docs: { iframeHeight: 550 },
     jest: ['sprk-card.component'],
   },
 };
 
+export const teaserIcon = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-card idString="teaser-icon">
+      <sprk-stack
+        sprkCardContent
+        itemSpacing="large"
+        sprkStackItem
+        additionalClasses="sprk-u-TextAlign--center"
+      >
+        <a
+          sprkLink
+          variant="unstyled"
+          href="#nogo"
+          sprkStackItem
+          analyticsString="teaser-icon-media"
+          class="sprk-u-AbsoluteCenter"
+        >
+          <sprk-icon
+            iconName="call-team-member"
+            additionalClasses="sprk-c-Icon--xl"
+          ></sprk-icon>
+        </a>
+
+        <h3 sprkHeading variant="displayFive" sprkStackItem>
+          Title
+        </h3>
+
+        <p sprkText variant="bodytwo" sprkStackItem>
+          Lorem ipsum dolor sit amet, doctus
+          invenirevix te. Facilisi perpetua an
+          pri, errem communemea at, mei prima
+          tantas signiferumque at.
+        </p>
+
+        <div sprkStackItem>
+          <a
+            sprkLink
+            variant="unstyled"
+            href="#nogo"
+            class="sprk-c-Button sprk-c-Button--secondary"
+            analyticsString="teaser-icon-cta"
+          >
+            Learn More
+          </a>
+        </div>
+      </sprk-stack>
+    </sprk-card>
+ `,
+});
+
+teaserIcon.story = {
+  name: 'Teaser With Icon',
+  parameters: {
+    docs: { iframeHeight: 550 },
+    jest: [
+      'sprk-card.component',
+      'sprk-card-content.directive',
+      'sprk-stack.component',
+      'sprk-stack-item.directive',
+      'sprk-text.directive',
+      'sprk-heading.directive',
+      'sprk-link.directive',
+      'sprk-icon.component',
+    ],
+  },
+};
+
+export const teaserWithDifferentElementOrder = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-card idString="teaser-diff-order">
+      <h3 sprkHeading sprkCardContent variant="displayFive" sprkStackItem>
```

Comment:
`<sprk-stack>`?

Reply:
@guern1kn, nope! Cause the card component adds the Stack for us.